docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -124,8 +124,8 @@ does not have an equivalent type.
 Decimal type handling
 ^^^^^^^^^^^^^^^^^^^^^
 
-``DECIMAL`` types with un-specified precision defaults to 38 and un-specified scale defaults to 0. These defaults can
-be overridden by setting the ``decimal-mapping`` configuration property or the ``decimal_mapping`` session property to
+``DECIMAL`` types with un-specified precision or scale is mapped to a Trino ``DECIMAL`` with a default precision of 38 and default scale of 0. These defaults can
+be changed by setting the ``decimal-mapping`` configuration property or the ``decimal_mapping`` session property to
```

Comment:
Also .. what you mean by 

"These can be changed" 

The scale can be changed? 

and how?

Reply:
See next sentence.

```
The scale of the resulting type is controlled via the ``decimal-default-scale``
configuration property or the ``decimal-rounding-mode`` session property. The precision is always 38.
```

Reply:
As the sentence stands now it is incorrect since  you are saying 

These defaults (implying default precision and default scale ) can be changed

but they you are saying precision can NOT be changed.. 

So I think it would be correct to say

The scale can be changed...

Reply:
Good point. Changing accordingly.